frontend/javascripts/oxalis/model/accessors/dataset_accessor.ts

```diff
@@ -827,3 +827,16 @@ export function getEffectiveIntensityRange(
 
   return layerConfiguration.intensityRange || defaultIntensityRange;
 }
+
+// Note that `hasSegmentIndex` needs to be loaded first (otherwise, the returned
+// value will be undefined). Dispatch an ensureSegmentIndexIsLoadedAction to make
+// sure this info is fetched.
+export function getMaybeSegmentIndexAvailability(
+  dataset: APIDataset,
+  layerName: string | undefined,
+) {
+  if (layerName == null || layerName.length < 1) {
```

Comment:
I think of a specific case, I just thought it would be smart to check that so that we dont need to iterate through the layers. but I think you're right, right now that shouldn't happen and if it does, the behaviour is still correct